RWFramework/RWFramework/Playlist/Playlist.swift

```diff
@@ -405,6 +405,16 @@ extension Playlist {
             $0.playNext(premature: true)
         }
     }
+    public func getCurrentAssets() -> [Asset] {
```

Comment:
Do you think we should be exposing the `Playlist` class and consider it user-facing, rather than hosting this method in `RWFramework`? Or if we keep this here, what is the division? Anything beyond simple `.play()`/`.pause()` has to be accessed by `RWFramework.playlist`? Just thoughts for the future; either is okay for now.